epictrack-api/src/api/schemas/request/staff_request.py

```diff
@@ -88,10 +95,15 @@ class StaffBodyParameterSchema(RequestBodyParameterSchema):
     )
 
     is_active = fields.Boolean(
-        metadata={"description": "Active status of the staff"},
-        required=True
+        metadata={"description": "Active status of the staff"}, required=True
     )
 
+    @pre_load
+    def convert_email_to_lower(self, data, **kwargs):  # pylint: disable=unused-argument
+        """Converts staff email into lower case string"""
+        data["email"] = data["email"].lower()
+        return data

```

Comment:
safer to check if email exists
 if 'email' in data:

I know email might be mandatory here , but that one day when people makes the email optional and foorgot to change this methiod , we will have the bug..so as a better practice